packages/eui/src/components/search_bar/query/default_syntax.ts

```diff
@@ -493,6 +499,7 @@ export const defaultSyntax: Syntax = Object.freeze({
     const dateFormat = options.dateFormat || defaultDateFormat;
     const parseDate = dateValueParser(dateFormat);
     const schema = options.schema || {};
+    const recognizedFields = schema.recognizedFields;
```

Comment:
Let's destructure this field out of the schema object, just in case it causes any issues in the parser (also, it's a slightly more succinct syntax):

```suggestion
    const { recognizedFields, ...schema } = options.schema || {};
```